pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1013,7 +1041,11 @@ bool GSQLBackend::get(DNSResourceRecord &r)
     } catch (SSqlException &e) {
       throw PDNSException("GSQLBackend get: "+e.txtReason());
     }
-    extractRecord(row, r);
+    try {
+      extractRecord(row, r);
+    } catch (...) {
+      goto skiprow;
```

Comment:
Yes, but for now silent ignoring is the safest thing to do. I'm not sure  an empty dnsname exception is caught down the line and i'm not aware of an other method to trigger a servfail from a gsql backend without destroying the backend.